configs/components/curl.rb

```diff
@@ -4,7 +4,14 @@
   pkg.url "http://buildsources.delivery.puppetlabs.net/curl-#{pkg.get_version}.tar.gz"
 
   pkg.build_requires "openssl"
-  pkg.build_requires "puppet-ca-bundle"
+  # TODO: remove the exception here once puppet-ca-bundle can be
+  # installed in cross-compiled environments
```

Comment:
We can fix this. Not in this PR but we can fix this. Can you please sign/date this comment?